configurations/operator/perf-regression-latency-and-throughput-multitenant-14-clients.yaml

```diff
@@ -0,0 +1,144 @@
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=10000001..15000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=15000001..20000000",
+], [
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..5000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=5000001..10000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=10000001..15000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=15000001..20000000",
+], [
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=5 -pop seq=1..7500000",
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=5 -pop seq=7500001..15000000",
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=5 -pop seq=15000001..22500000",
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=5 -pop seq=22500001..30000000",
+], [
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=5 -pop seq=1..7500000",
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=5 -pop seq=7500001..15000000",
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=5 -pop seq=15000001..22500000",
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=5 -pop seq=22500001..30000000",
+], [
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=5 -pop seq=1..7500000",
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=5 -pop seq=7500001..15000000",
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=5 -pop seq=15000001..22500000",
+    "cassandra-stress write no-warmup cl=QUORUM n=7500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=5 -pop seq=22500001..30000000",
+], [
```

Comment:
Why these writes during prepare are different?
(schem is different, size, and consistency is QUORUM).

Reply:
I don't get the idea of your question because here we have identical prepare-write commands. `QUORUM` CL is what is used in the original config

Reply:
The first chunk of writes are different from this chunk.

Reply:
Ok, now your question is clear.
This particular config contains 4 latency clusters, 3 thrpt ones, then repeat again 4 and 3 the same way.

Reply:
please change cl to ALL

Reply:
I changed only load numbers. CL stays the same as in the original throughput test configuration.
Do you want to say that it was incorrect all the time for throughput tests?

Reply:
> please change cl to ALL

Done